### ISSUE TITLE
Error occured when deleting current active fit

### ISSUE THREAD

**Issue by Nomadmonk** [pyfa-org/Pyfa#1221]
```OS version: 	Windows-8-6.2.9200
Python: 	2.7.10
wxPython: 	3.0.2.0
SQLAlchemy: 	1.0.5
Logbook: 	1.0.0
pyfa version: 1.29.3 Stable - YC119.5 1.0
pyfa root: C:\Program Files (x86)\pyfa
save path: C:\Users\Sebastien\.pyfa
fs encoding: mbcs

EXCEPTION: A conflicting state is already present in the identity map for key (<class 'eos.saveddata.fit.Fit'>, (37,))

  File "C:\Program Files (x86)\pyfa\library.zip\gui\sfBrowserItem.py", line 317, in OnLeftUp
    btn.DoCallback()
  File "C:\Program Files (x86)\pyfa\library.zip\gui\sfBrowserItem.py", line 50, in DoCallback
    self.callback()
  File "C:\Program Files (x86)\pyfa\library.zip\gui\shipBrowser.py", line 1802, in deleteBtnCB
    self.deleteFit()
  File "C:\Program Files (x86)\pyfa\library.zip\gui\shipBrowser.py", line 1819, in deleteFit
    sFit.deleteFit(self.fitID)
  File "C:\Program Files (x86)\pyfa\library.zip\service\fit.py", line 188, in deleteFit
    eos.db.saveddata_session.delete(fit)
  File "C:\python-2.7.10\lib\site-packages\sqlalchemy\orm\session.py", line 1632, in delete
  File "C:\python-2.7.10\lib\site-packages\sqlalchemy\orm\identity.py", line 132, in add
```

**Comment by blitzmann** [comment]
See #1207 for a build that should include a fix for this. I was waiting until the end of this week to see if any other bugs popped up before creating a new point release, but will try to get one out tonight / tomorrow. :)

**Comment by blitzmann** [comment]
Closing this due to lack of activity, and possibly fixed in #1207. if you are having any other issues, please feel free to open another issue 😄
